cf/api/resources/service_instances_test.go

```diff
@@ -125,7 +127,127 @@ var _ = Describe("ServiceInstanceResource", func() {
 				Expect(fields.LastOperation.Type).To(Equal("create"))
 				Expect(fields.LastOperation.State).To(Equal("in progress"))
 				Expect(fields.LastOperation.Description).To(Equal("fake state description"))
+				Expect(fields.LastOperation.CreatedAt).To(Equal("fake created at"))
+				Expect(fields.LastOperation.UpdatedAt).To(Equal("fake updated at"))
 			})
+
+			Context("When created_at is null", func() {
+				It("unmarshalls the service instance resource model", func() {
+					var resourceWithNullCreatedAt ServiceInstanceResource
+
+					err := json.Unmarshal([]byte(`
+						{
+							"metadata": {
```

Comment:
This seems very noisy when inline. Did you consider defining once and modifying string (using `strings` package for instance) later for each case. Especially since you also need to do the same for `updated_at` ?